api/cloud_api/util/S3BucketApiHandler.js

```diff
@@ -11,6 +11,10 @@ class S3BucketApiHandler {
   }
   getSignedUrl(s3Params, bucket) {
     return new Promise((resolve, reject) => {
+      if(s3BucketKeys.AWSACCESSKEYID == 'NOT_SET'
+      && s3BucketKeys.AWSSECRETKEY == 'NOT_SET') {
+        return resolve(true);
+      }
```

Comment:
@allen-tran we should use this class for writing data to s3 for printer

Reply:
ok